lib/WebpackConfig.js

```diff
@@ -115,7 +115,14 @@ class WebpackConfig {
         // Features/Loaders options
         this.copyFilesConfigs = [];
         this.sassOptions = {
-            resolveUrlLoader: true
+            resolveUrlLoader: true,
+            resolveUrlLoaderOptions: {
+                engine: 'postcss',
```

Comment:
Is there a reason why you explicitely set the value of those options there?
Couldn't we go with `resolveUrlLoaderOptions: {}` instead?